src/Middlewares/Guzzle/GenerateSignature.php

```diff
@@ -15,14 +16,14 @@ class GenerateSignature
     protected $generator;
 
     /**
-     * Expect an instance of the generator so we can sign the request
+     * Expect a configuration to build our generator with.
      *
-     * @param \SoapBox\SignedRequests\Requests\Generator $generator
-     *        An instance of the signed request generator
+     * @param \SoapBox\SignedRequests\Configurations\Configuration $configuration
+     *        The configuration to use for generating signed requests.
      */
-    public function __construct(Generator $generator)
+    public function __construct(Configuration $configuration)
```

Comment:
I feel like this is a breaking change on any dependent projects so we should update to `v0.2`